quick-github-guide.md

```diff
@@ -0,0 +1,163 @@
+  Make your updates to the content.
+
+1. Click **Preview changes** at the top of the editing area to see the effect of your changes.
+
+1. If you need to make more changes, click **Edit file** at the top of the preview area.
+
+1. When you are ready to submit your changes, scroll down to the 
+  **Propose file change**
+   section at the bottom of the editing area.
+
+      * Enter a short description of your update. This short description becomes the 
+        title of your pull request (PR).
+
+      * In the second text box (for the extended description), enter a more detailed
+        description.
+
+1. Click **Propose file change**. A new screen appears, offering you the 
+  opportunity to open a pull request.
+
+1. Click **Create pull request**. 
+
+1. Optionally, edit the pull request title and description.
+
+1. Make sure **Allow edits from maintainers** remains checked.
+
+1. Click **Create pull request** again. You have now sent a request to the repository
+  maintainers to review your change.
+
+1. Check the online preview of your changes:
+
+      * Wait for the automated PR workflow to do some checks. When it's ready,
+        you should see a comment like this: **deploy/netlify — Deploy preview ready!**
+      * Click **Details** to the right of "Deploy preview ready" to see a preview
+        of your updates.
+
+## Using the command line
+
+Here's a quick guide to a fairly standard GitHub workflow using git on the command line:
+
+1. Fork the kubeflow/website repo:
+
+    * Go to the [kubeflow/website repo](https://github.com/kubeflow/website) on GitHub.
+    * Click **Fork** to make your own copy of the repo. GitHub creates a copy
+      at `https://github.com/<your-github-username>/website`.
+
+1. Open a command window on your local machine.
+
+1. Clone your forked repo, to copy the files down to your local machine.
+  This example creates a directory called `kubeflow` and uses SSH cloning to
+  download the files:
+
+    ```
+    mkdir kubeflow
+    cd kubeflow/
+    git clone git@github.com:<your-github-username>/website.git
+    cd website/
+    ```
+
+1. Add the upstream repo as a git remote repo:
+
+    ```
+    git remote add upstream https://github.com/kubeflow/website.git
+    ```
+
+1. Check your remotes:
+
+    ```
+    git remote -vv
+    ```
+
+    You should have 2 remote repos:
+
+      -  `origin` - points to your own fork of the repo on gitHub -
+         that is, the one you cloned my local repo from.
+      -  `upstream` - points to the actual repo on gitHub.
+
+1. Create a branch. In this example, replace `doc-updates` with any branch name
+  you like. Choose a branch name that helps you recognise the updates you plan
+  to make in that branch:
+
+    ```
+    git checkout -b doc-updates
+    ```
+
+1. Add and edit the files as you like. The doc pages are in the
+  `/website/content/docs/` directory.
+
+1. Run `git status` at any time, to check the status of your local files.
```

Comment:
Suggestion: mention the goal before the instruction

To check the status of your local files, run `git status` at any time.